webapp/src/main/java/org/opentestsystem/rdw/admin/repository/GroupImportRepository.java

```diff
@@ -0,0 +1,24 @@
+package org.opentestsystem.rdw.admin.repository;
+
+/**
+ * This repository is responsible for working with imports.
+ */
+public interface GroupImportRepository {
+
+    /**
+     * Create a Import record to represent a warehouse
+     * to data-mart Group migration.
+     *
+     * @param digest    The import digest (max 32-character string)
+     * @param batch     The import batch (max 250-character string)
+     * @return The created import id
+     */
+    long create(String digest, String batch);
+
+    /**
+     * Trigger the given Import.
+     *
+     * @param importId The import id
+     */
+    void trigger(long importId);
```

Comment:
nit: the verb `trigger` seems loaded. I'm assuming all this does is mark the import record as PROCESSED? Why not a method name that reflects that?